tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/ReleasePlanDetails.cs

```diff
@@ -33,6 +33,46 @@ public class ReleasePlanDetails
         public string SDKLanguages { get; set; } = string.Empty;
         public bool IsSpecApproved { get; set; } = false;
         public int ApiSpecWorkItemId { get; set; } = 0;
+        public override string ToString()
+        {
+            var sdkInfoFormatted = SDKInfo != null && SDKInfo.Any()
+                ? string.Join("\n    ", SDKInfo.Select(s => s.ToString()))
+                : "None";
+
+            var specPullRequestsFormatted = SpecPullRequests != null && SpecPullRequests.Any()
+                ? string.Join(", ", SpecPullRequests)
+                : "None";
+
+            return $@"ReleasePlanDetails:
```

Comment:
It's probably better to just return json at this point, that way we don't get potential drift if new properties are added to the class but not to `ToString()`